copy_test.go

```diff
@@ -1779,7 +1779,9 @@ func TestCopyGraph_WithOptions(t *testing.T) {
 	t.Run("MountFrom error", func(t *testing.T) {
 		root = descs[6]
 		dst := &countingStorage{storage: cas.NewMemory()}
-		opts = oras.CopyGraphOptions{}
+		opts = oras.CopyGraphOptions{
+			Concurrency: 1,
+		}
```

Comment:
What is the consideration?

Reply:
We want to make the run result deterministic. So we want to make sure that only one goroutine is run at a time, basically make it sequential.

Reply:
Got it. Can you add this explanation to the comment so that others looking at the code can understand?

Reply:
Added a comment